stable/README.md

```diff
@@ -37,37 +40,33 @@ $ mv tiller /usr/local/bin
 If you're running on Mac, the curl commands above would be:
 
 ```bash
-$ curl -s https://storage.googleapis.com/kubernetes-helm/helm-v2.14.1-darwin-amd64.tar.gz | tar xz
+$ curl -s https://storage.googleapis.com/kubernetes-helm/helm-v2.16.1-darwin-amd64.tar.gz | tar xz
 $ cd darwin-amd64
 $ mv helm /usr/local/bin
 $ mv tiller /usr/local/bin
 ```
 
-The following command will configure the current users environment for Helm; it will create a `.helm` directory in `${HOME}`.
-
-```bash
-$ helm init --client-only
-$HELM_HOME has been configured at /home/clusteradmin/.helm.
-Not installing Tiller due to 'client-only' flag having been set
-Happy Helming!
-```
-
-Alternatively, to keep isolated between individuals sharing a login, you can have separate Kube, Helm, and Tiller state (so run these before the above init command):
+To keep isolated between individuals sharing a login, you can have separate Kubernetes, Helm, and Tiller state. Run these before proceeding.
 
 ```bash
 export TILLER_NAMESPACE=nuodb
 export HELM_HOME=`pwd`/.helm
 export KUBECONFIG=`pwd`/.kube/config
```

Comment:
this change is wrong.

The default install is `helm init --client-only`.

The one you kept is the fallback that is usually not required.

Reply:
I didn't change the text here, only moved the existing text "export commands" earlier since they had been referenced above, but not yet created.